### PR TITLE
refactor(shared-data): update load names for h/s labware+adapter definitions

### LINKED ISSUE #10938: Update H/S definition loadNames
# Overview

The heater shaker adapter definitions need updated loadNames.

# Acceptance criteria

The API loadNames are corrected to the following: 

- [ ] opentrons_96_deep_well_adapter_nest_wellplate_2ml_deep
- [ ] opentrons_96_flat_bottom_adapter_nest_wellplate_200ul_flat
- [ ] opentrons_96_pcr_adapter_nest_wellplate_100ul_pcr_full_skirt
- [ ] opentrons_universal_flat_adapter_corning_384_wellplate_112ul_flat
- [ ] opentrons_96_pcr_adapter_armadillo_wellplate_200ul

### LINKED ISSUE #10973: h/s: update display names for adapter definitions
# Overview

The 5 new h/s definition display names need to be updated

# Acceptance criteria

- [x] Opentrons 96 Deep Well Adapter with NEST Deep Well Plate 2 mL
- [ ] Opentrons 96 PCR Adapter with Armadillo Well Plate 200 µl
- [x] Opentrons 96 PCR Adapter with NEST Well Plate 100 µl
- [x] Opentrons 96 Flat Bottom Adapter with NEST 96 Well Plate 200 µL Flat
- [x] Opentrons Universal *Flat* Adapter with Corning 384 Well Plate 112 µL Flat

### PR DESCRIPTION
**Pull request by TamarZanzouri**
# Overview

closes #10938.
closes #10973.
updated loadNames to suggested names.

# Changelog

- Updated file names in 'shared-data/labware'
- Updated loadModule key
- Updated ts getAdapterName and test

# Review requests

1. did I miss anything? 
2. ts changes look correct? 
3. test on a robot?

# Risk assessment

Medium. changed file names and loadName. could cause adapters not to load if there is a mismatch in names.

### CODE REVIEW

**Review by skyewindsor — APPROVED** [review]
The new loadNames look good to me. 
I cannot speak into the ts changes or testing on a robot though

**Review by sanni-t — COMMENTED** [review]
Looks good. Let's also update the file names to reflect the updated `loadName`s

**Comment by skyewindsor** [comment]
@TamarZanzouri The `"displayName"`s will need to be updated too. Ticket here: https://github.com/Opentrons/opentrons/issues/10973

**Comment by TamarZanzouri** [comment]
> @TamarZanzouri The `"displayName"`s will need to be updated too. Ticket here: #10973

updated :-)

File: shared-data/labware/definitions/2/opentrons_universal_flat_adapter_corning_384_wellplate_112ul_flat/1.json [hunk 1]
```diff
@@ -438,7 +438,7 @@
     "brandId": []
   },
   "metadata": {
-    "displayName": "Opentrons Flat Plate Adapter with Corning 384 Well Plate 112uL",
+    "displayName": "Opentrons Universal Adapter with Corning 384 Well Plate 112 µL Flat",
```
Comment (sanni-t) [inline shared-data/labware/definitions/2/opentrons_universal_flat_adapter_corning_384_wellplate_112ul_flat/1.json]: @skyewindsor should this be "Opentrons Universal Flat Adapter..." to match the load name?

File: shared-data/labware/definitions/2/opentrons_96_deep_well_adapter_nest_wellplate_2ml_deep/1.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "links": []
   },
   "metadata": {
-    "displayName": "Opentrons 96 Deep Well Adapter with Nest Deep Well Plate 2ml",
+    "displayName": "Opentrons 96 Deep Well Adapter with NEST Deep Well Plate 2mL",
```
Comment (mcous) [inline shared-data/labware/definitions/2/opentrons_96_deep_well_adapter_nest_wellplate_2ml_deep/1.json]:
```suggestion
    "displayName": "Opentrons 96 Deep Well Adapter with NEST Deep Well Plate 2 mL",
```

File: shared-data/labware/definitions/2/opentrons_96_pcr_adapter_nest_wellplate_100ul_pcr_full_skirt/1.json [hunk 3]
```diff
@@ -19,7 +19,7 @@
     "links": []
   },
   "metadata": {
-    "displayName": "Opentrons 96 PCR Plate Adapter with Nest Well Plate 100uL PCR Full Skirt",
+    "displayName": "Opentrons 96 PCR Adapter with NEST Well Plate 100ul",
```
Comment (mcous) [inline shared-data/labware/definitions/2/opentrons_96_pcr_adapter_nest_wellplate_100ul_pcr_full_skirt/1.json]:
```suggestion
    "displayName": "Opentrons 96 PCR Adapter with NEST Well Plate 100 µl",
```

**Review by mcous — CHANGES_REQUESTED** [review]
- Need space between number and unit in display names to match Opentrons style
- Need to use `µ` in display names, not `u`

File: shared-data/labware/definitions/2/opentrons_universal_flat_adapter_corning_384_wellplate_112ul_flat/1.json (hunk 1 above)
Comment (skyewindsor) [reply shared-data/labware/definitions/2/opentrons_universal_flat_adapter_corning_384_wellplate_112ul_flat/1.json]: @sanni-t Yes, that display name should say "Opentrons Universal Flat Adapter with Corning 384 Well Plate 112 µl Flat". @TamarZanzouri are you able to update that? I made a mistake in the ticket

Comment (TamarZanzouri) [reply shared-data/labware/definitions/2/opentrons_universal_flat_adapter_corning_384_wellplate_112ul_flat/1.json]: @sanni-t nice catch! @skyewindsor please confirm so I can make the change

Comment (skyewindsor) [reply shared-data/labware/definitions/2/opentrons_universal_flat_adapter_corning_384_wellplate_112ul_flat/1.json]: @TamarZanzouri confirming you can make this change!

**Review by mcous — APPROVED** [review]
Changes LGTM, once the change in the thread above is resolved

**Review by sanni-t — APPROVED** [review]
LGTM! Thanks for making the changes. Good to merge once CI clears

**Pull request merged**